game-core/src/main/java/games/strategy/engine/framework/lookandfeel/LookAndFeel.java

```diff
@@ -41,11 +42,24 @@
 import games.strategy.debug.ClientLogger;
 import games.strategy.engine.framework.system.SystemProperties;
 import games.strategy.triplea.settings.ClientSetting;
+import games.strategy.triplea.settings.SettingsWindow;
 
 /**
  * Provides methods for working with the Swing Look-And-Feel.
  */
 public final class LookAndFeel {
+  static {
+    ClientSetting.LOOK_AND_FEEL_PREF.addSaveListener(newValue -> {
+      setupLookAndFeel(newValue);
+      SettingsWindow.updateLookAndFeel();
+      JOptionPane.showMessageDialog(
+          null,
+          "WARNING: look and feel changes can cause instability. Please restart all running TripleA instances",
```

Comment:
Nit: Missing a period at the end of the message.  Only flagging because the intention appears to have been to use two sentences due to the period after _instability_.

Reply:
FWIW - Intent, was for minimal punctuation. Given we have two sentences, without a line break we need a period there : )

We could probably use a style guide; whether to prefer the final period or not. I'm also noticing that we have all our 'yes' buttons on the right. This is starting to seem wrong to me, for better or worse it does seem like default, positive menu selection is on left. 

Getting back to this topic, I tweaked this message a bit, no need to say 'WARNING' when in a warning dialog already, and I broke up to two line and made the trailing period consistent:

91ea813a4

Reply:
Getting back to this topic, I tweaked this message a bit, no need to say 'WARNING' when in a warning dialog already, and I broke up to two line and made the trailing period consistent:

91ea813a4